### DISCUSSION TITLE
Is it possible to batch `router.completion()`?

### DISCUSSION THREAD

**Discussion by AAnirudh07** [BerriAI/litellm#3753] [Q&A] [answered]
Hello, 
I was going through the documentation for Routers and noticed that it supports completion: [router.completion](https://litellm.vercel.app/docs/routing#available-endpoints). Is it possible to use this completion function in `batch_completion`? I noticed that there is a similar discussion [here](https://github.com/BerriAI/litellm/discussions/3541), but it looks like the mentioned answer didn't work for the author, so I was wondering if we have to do something different. Any help would be greatly appreciated!

**Comment by ishaan-jaff** [comment, marked as answer]
Live here: https://github.com/BerriAI/litellm/pull/3751 

@AAnirudh07 can we hop on a call sometime this week to learn how we can improve litellm for you ? What's the best email to send an invite to ? 


If it's easier here's my cal: https://calendly.com/d/4mp-gd3-k5k/litellm-1-1-onboarding-chat
